docs/field-values-usage.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[ecs-using-the-categorization-fields]]
+=== Using the Categorization Fields
+
+The event categorization fields work together to identify and group similar events from multiple data sources.
+
+These general principles can help guide the categorization process:
+
+* Events from multiple data sources that are similar enough to be viewed or analyzed together, should fall into the same `event.category` field.
+* Both `event.category` and `event.type` may be populated with multiple allowed values, if the event can be reasonably classified into more than one category and/or subcategory.
```

Comment:
* Both `event.category` and `event.type` are arrays and may be populated with multiple allowed values, if the event can be reasonably classified into more than one category and/or type.
* `event.kind`, `event.category`, `event.type` and `event.outcome` all have allowed values, this is to normalize these fields.  Values that aren't in the list of allowed values should not be used.